### PR TITLE
added Slicing riddle

### PR DESCRIPTION
**Pull request by grolimur**
added an exercise on slicing based on Harry Potter and the Chamber of Secrets

### CODE REVIEW

**Comment by alee** [comment]
thanks for taking the time to submit this PR! This is a cute example (and I enjoy Harry Potter) but I don't find this to be a great example for slicing; I find it a bit hard to follow with all the string indexing going on. 

I'm open to other @swcarpentry/python-novice-gapminder-maintainers or community thoughts though...

**Comment by grolimur** [comment]
I guess you are right: the example is a little bit complicated.
Thanks for your (nice) feedback.

**Comment by alee** [comment]
Closing in advance of a May deadline to convert to the new Workbench lesson - please reopen there if you'd like us to consider another slicing example. Cheers!
